src/darwin/Framework/CHIPTests/MTRControllerTests.m

```diff
@@ -62,6 +62,10 @@ - (void)testFactoryLifecycle
     XCTAssertTrue([factory startControllerFactory:factoryParams error:nil]);
     XCTAssertTrue([factory isRunning]);
 
+    // Starting again with identical params is a no-op
```

Comment:
It's a no-op even with different params, no?

Reply:
At the moment it is, but I think we may want to make that return an INVALID_STATE error instead?

Reply:
Yeah, that's fair.